components/DatePick.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { SetStateAction, Dispatch } from "react";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import styled from "styled-components";
+
+type DatePickProps = {
+  dates: string[];
+  onSetDates?: Dispatch<SetStateAction<string[]>>;
+  className?: string;
+  inline: boolean;
+};
+
+export default function DatePick({
+  dates,
+  onSetDates,
+  className,
+  inline,
+}: DatePickProps) {
+  const datum: Array<Date> = [];
+
+  dates.forEach((date) => {
+    datum.push(new Date(date));
+  });
+
+  function handleDate(date: Date): void {
+    const d: string = date.toDateString();
```

Comment:
```suggestion
    const dateString: string = date.toDateString();
```

d is very bad naming ;)

Reply:
I really can't deny that! I will come up with something better!